source/hic_hal/target_config.h

```diff
@@ -18,16 +18,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+/*******************************************************************************
+* Copyright 2019, Cypress Semiconductor Corporation
+* or a subsidiary of Cypress Semiconductor Corporation.
+*
+* Description of changes made by Cypress.
+*
+* - added flash write operation with dynamic switching of flashloaders for PSoC6 targets
+*
+*****************************************************************************/
 #ifndef TARGET_CONFIG_H
 #define TARGET_CONFIG_H
 
 #include "stddef.h"
 #include "stdint.h"
+#include "stdbool.h"
  
 #include "flash_blob.h"
 #include "macro.h"
 
+#ifdef TARGET_MCU_PSOC6
```

Comment:
Please remove target-specific ifdef'd code from this file. Looks like the changes can be made common.

Reply:
target-specific ifdef'd code has been removed from this file.